packages/dappmanager/src/modules/ethClient/apiUrl.ts

```diff
@@ -38,19 +40,21 @@ export async function getEthConsClientApiUrl(dnpName: string): Promise<string> {
     dnp.chain.serviceName
   ) {
     port = dnp.chain.portNumber;
-    domain = getPrivateNetworkAlias({
+    domain = determineNetworkAlias({
```

Comment:
You could also call here `build/generateNetworkAliasByPkg({service: string, pkg: InstalledPackageData})`

Reply:
`dnp.chain.serviceName` does not seem 100% reliable either